Dockerfile

```diff
@@ -25,9 +25,10 @@ RUN apt-get update && \
 # Build and install scamper
 RUN ls -l
 RUN mkdir /scamper-src
-ADD ./vendor/scamper/ /scamper-src
-RUN chmod +x /scamper-src/scamper-cvs-20190916/configure
-WORKDIR /scamper-src/scamper-cvs-20190916/
+ADD ./vendor/ /scamper-src/
```

Comment:
Could you make the directory and then add only the tarball?  As written, this also adds libparistraceroute to the /scamper-src directory, which is not some thing we want.